src/graphConfig.ts

```diff
@@ -115,6 +115,9 @@ export type TGraphConstants = {
     DEFAULT_Z_INDEX: number;
     THRESHOLD_LINE_HIT: number;
     MIN_ZOOM_FOR_CONNECTION_ARROW_AND_LABEL: number;
+    LABEL: {
+      INNER_PADDING: number;
```

Comment:
Thank you for your contribution. I would like to ask you to change the config `LABEL.INNER_PADDING` to  `[number, number, number, number]`, which will specify the padding for each side.

Reply:
done https://github.com/gravity-ui/graph/pull/27/commits/740a1c8e62f7ef87ce5f2dab7ad999e520cb32dd